src/views/AverageWaveforms/AverageWaveformsViewData.ts

```diff
@@ -20,13 +21,15 @@ export type AverageWaveformsViewData = {
     averageWaveforms: AverageWaveformData[]
     samplingFrequency: number
     noiseLevel: number
+    channelLocations?: {[key: string]: number[]}
```

Comment:
Could this be anything other than 2d?

Reply:
It could be 3d, but we're only displaying 2d.